### DISCUSSION TITLE
Min doesn't show as an app in Windows

### DISCUSSION THREAD

**Discussion by JohnHBrady** [minbrowser/min#1864] [Ideas]
Min when installed doesn't show up on Windows app list. This makes it hard for me to do some things like pick Min as the place to open images. Can I get around this? Why does this happen?

**Comment by PalmerAL** [comment]
It does for me (it's missing the logo though, which I think #1781 might fix if it ever gets merged):

<img width="538" alt="Screen Shot 2022-01-11 at 1 40 33 PM" src="https://user-images.githubusercontent.com/10314059/149010190-3aceb7a8-50df-4327-b167-3b87c436b1b7.png">

If it's not showing up for you there, perhaps reinstalling would work?

Min also doesn't show up in the photo viewer list for me, I think because we don't indicate that we can open photos. That would be a good idea to fix.

**Comment by JohnHBrady** [comment]
I figured out that I can open photos and PDFs by dragging to the tab bar. Nice, and maybe should be mentioned in the tour or elsewhere?

**Reply by PalmerAL** [reply]
Yeah, good point. Really, we should just fix the list of file extensions that Min says it supports so you can open files from Explorer.
